src/components/microlearning/styles/VideoDetailPage.scss

```diff
@@ -0,0 +1,96 @@
+.video-detail-page-wrapper {
```

Comment:
The use of CSS is discouraged, you should use paragon utility classes as much as possible. The reason the use is discouraged is because of themes. CSS would cause issues with the UI for different themes. 

I will try to add as much example to replace usage of direct CSS here as I can remember. But I think @adamstankiewicz would be a better person to review this as well.

Reply:
Here is the link to utility classes docs https://paragon-openedx.netlify.app/foundations/css-utilities/

Reply:
I adding examples for some cases, but you should do the same for all other cases as well.

Reply:
@saleem-latif Custom CSS is necessary for some places because we are using a custom video.js plugin -` videojs-vjstranscribe`. The elements of this plugin need to be customized to cater to the hidden classes and other specific requirements of the design. Other than that, I have tried to replace custom CSS with utility classes.